netbout-client/pom.xml

```diff
@@ -178,11 +178,6 @@
                     <plugin>
                         <groupId>com.qulice</groupId>
                         <artifactId>qulice-maven-plugin</artifactId>
-                        <configuration>
-                            <excludes>
-                                <exclude>findbugs:.*</exclude>
```

Comment:
@exper0  right now we have whole project excluded from the FindBugs checking `<exclude>findbugs:.*</exclude>`
you have to fix some files and and have to set exclusions for others.
then create a new puzzle here to fix other files and remove them from the exclusions